### PR TITLE
udpated description of X-Tenant-ID header field

### LINKED ISSUE #320: X-Tenant-ID: reword the description
From [183](http://zalando.github.io/restful-api-guidelines/#183):

> The tenant id for future platform multitenancy support. Should not be used unless new platform multitenancy is truly supported. But should be used by New Platform Prototyping services. Must be validated for external retailer, supplier, etc. tenant users via OAuth2; details in clarification. Currently only used by New Platform Prototyping services.

We need to reword this really old text.

### PR DESCRIPTION
**Pull request by codekultur**
fixes #320

### CODE REVIEW

File: chapters/proprietary-headers.adoc [hunk 1]
```diff
@@ -57,12 +57,12 @@ spaces, NULL). If a legacy subsystem can only work with flow IDs of a
...
-support. _Should not be used unless new platform multitenancy is truly
-supported. But should be used by New Platform Prototyping services._
-Must be validated for external retailer, supplier, etc. tenant users via
-OAuth2; details in clarification. Currently only used by New Platform
-Prototyping services. |9f8b3ca3-4be5-436c-a847-9cd55460c495
+|X-Tenant-ID |String | Identifies the tenant initiated the request
+to the multi tenant Zalando Platform. The X-Tenant-ID must be set 
+according to the Business Partner ID extracted from the OAuth token when 
+a request from a Business Partner hits the Zalando Platform and must be 
+passed on in subsequent calls inside the Platform.
```
Comment (maxim-tschumak) [inline chapters/proprietary-headers.adoc]: > [..] must be passed on in subsequent calls inside the Platform.

Hint: This is already explicitly stated in [184] for all proprietary headers.

Comment (codekultur) [reply chapters/proprietary-headers.adoc]: Thx.. Good spot! Remove the subsentence...

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Pull request merged**